src/menv/commands/create.py

```
@@ -54,6 +54,85 @@
+    # Future extensions can be added here
+    # {
+    #     "tag": "ollama-models",
+    #     "name": "LLM Models",
+    #     "description": "Download large LLM models for Ollama (Requires heavy network usage)",
+    #     "profile_specific": False,
+    # },
+]
+
+
+# Data structure for optional tasks
+class OptionalTask(TypedDict):
```

Ruff (0.14.10)

Redefinition of unused `OptionalTask` from line 58

(F811)